src/PrestaShopBundle/Resources/config/services/core/form/command_builder.yml

```diff
@@ -150,7 +150,10 @@ services:
     arguments:
       - !php/const PrestaShopBundle\Form\Admin\Extension\ModifyAllShopsExtension::MODIFY_ALL_SHOPS_PREFIX
 
-  prestashop.core.form.command_builder.product.combination.update_combination_commands_builder:
-    class: PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\Combination\UpdateCombinationCommandsBuilder
+  PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\Combination\UpdateCombinationCommandsBuilder: ~
+    # @todo: tags commented bellow, to avoid affecting combination page until all the related commands are refactored.
+    # tags: [ 'core.combination_command_builder' ]
+
+  PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\Combination\CombinationStockAvailableCommandsBuilder: ~
```

Comment:
Need autowire true?

Reply:
Same as the other, no arguments so autowiring not needed for now